.gitignore

```diff
@@ -6,6 +6,16 @@ __pycache__/
 # C extensions
 *.so
 
+# csv and exel files
+*.csv
+*.xlsx
+
+#testing jsons
+test_*.json
```

Comment:
> Are these still needed? Is the debug code still used?

Since we're excluding the .json that I previously included in the repo, I will go ahead and remove this. Would you like me to make an example.json for the repo in the same way we have an example.config currently?